### PR TITLE
DEP: Stop adding parameters into the object metadata

### PR DESCRIPTION
**Pull request by tnatt**
PR to stop adding parameters into every object metadata. 

Parameters are uploaded via the `fmu-sumo-uploader` as one file per realization with `content="parameters",` and is planned to be added to the new `Realization` object #774 -> hence there is no longer a need to include it in the object metadata

Adresses #345

### CODE REVIEW

File: tests/test_units/test_dictionary.py [hunk 1]
```diff
@@ -145,7 +131,7 @@ def test_invalid_dict(
 def test_read_parameters_txt():
     with NamedTemporaryFile() as tf:
         tf.write(
-            b"""SENSNAME rms_seed
+            b"""SENSNAME 'rms seed'
```
Comment (tnatt) [inline tests/test_units/test_dictionary.py:134]: Just wanted to test that `shlex.split()` handled spaces in parameter names

File: src/fmu/dataio/_utils.py [hunk 2]
```diff
@@ -208,71 +209,28 @@ def uuid_from_string(string: str) -> uuid.UUID:
...
-    dict_str_to_str = _design_kw.extract_key_value(parameterlines)
-    return {key: check_if_number(value) for key, value in dict_str_to_str.items()}
+    with open(pfile) as f:
+        for line in f:
+            line_parts = shlex.split(line)
+            if len(line_parts) == 2:
+                key, value = line_parts
+                res[key] = check_if_number(value)
+            else:
+                raise ValueError(f"More or less than two items found in line {line}")
```
Comment (jcrivenaes) [inline src/fmu/dataio/_utils.py:231]: Perhaps the error message should be a bit more clear on context?

**Comment by perolavsvendsen** [comment]
I _think_ this should be OK, but I recommend asking around a bit if anyone is actually depending on this before merging. I am pretty sure REP is not using it, but I don't know about Webviz and any other consumer.

**Comment by tnatt** [comment]
> I _think_ this should be OK, but I recommend asking around a bit if anyone is actually depending on this before merging. I am pretty sure REP is not using it, but I don't know about Webviz and any other consumer.

I have asked a bit around:
DynaGeo does not use ERT parameters per today, and 👍  from Webviz to remove them.
Any other consumers we need to confer before doing the change?

**Pull request merged**